src/main/java/net/nuagenetworks/bambou/RestObject.java

```diff
@@ -212,16 +212,21 @@ public void instantiateChild(RestObject childRestObj, RestObject fromTemplate, I
     }
 
     @Override
-    public void assign(List<? extends RestObject> childRestObjs) throws RestException {
+    public void assign(List<? extends RestObject> childRestObjs, Class<? extends RestObject> objectType) throws RestException {
```

Comment:
For API backwards compatibility, the following method signature should still be supported:

public void assign(List<? extends RestObject> childRestObjs) throws RestException